src/main/java/uk/gov/gchq/magmacore/util/DataObjectUtils.java

```diff
@@ -14,29 +14,13 @@
 
 package uk.gov.gchq.magmacore.util;
 
-import java.time.LocalDateTime;
 import java.util.UUID;
 
-import uk.gov.gchq.hqdm.iri.HQDM;
-import uk.gov.gchq.hqdm.iri.IRI;
-import uk.gov.gchq.hqdm.iri.IriBase;
-import uk.gov.gchq.hqdm.model.PointInTime;
-import uk.gov.gchq.hqdm.model.PossibleWorld;
-import uk.gov.gchq.hqdm.model.impl.PointInTimeImpl;
-
 /**
  * Utilities for building and generating HQDM objects.
  */
 public final class DataObjectUtils {
```

Comment:
Renamed this class and updated the references since the UID class did not exist any more. Also had to update the checkstyle suppressions to allow the UID name.